vendor/browan/doorwindowsensor-codec.yaml

```diff
@@ -0,0 +1,32 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: doorwindowsensor.js
+  # Examples (optional)
+  format:
+    - description: door window sensor
+      input:
+        fPort: 100
+        bytes: [0, 7]
+      output:
+        data:
+          Status[0]:           
```

Comment:
Same as above. This is an example output, based on your example input. The validator will use this example to test your codecs.